templates/wordpress-vanilla/files/wp-config.php

```diff
@@ -0,0 +1,98 @@
+<?php
+
+// Set default scheme and hostname.
+$site_scheme = 'http';
+$site_host = 'localhost';
+
+// Update scheme and hostname for the requested page.
+if (isset($_SERVER['HTTP_HOST'])) {
+  $site_host = $_SERVER['HTTP_HOST'];
+  $site_scheme = !empty($_SERVER['HTTPS']) ? 'https' : 'http';
+}
+
+if (getenv('PLATFORM_RELATIONSHIPS')) {
+	if(getenv('DB_NAME')){
+
+		// Avoid PHP notices on CLI requests.
+		if (php_sapi_name() === 'cli') {
+			session_save_path("/tmp");
+		}
+
+		// We are using the first relationship called "database" found in your
+		// relationships. Note that you can call this relationship as you wish
+		// in your `.platform.app.yaml` file, but 'database' is a good name.
+		define( 'DB_NAME', getenv('DB_NAME'));
+		define( 'DB_USER', getenv('DB_USER'));
+		define( 'DB_PASSWORD', getenv('DB_PASSWORD'));
+		define( 'DB_HOST', getenv('DB_HOST'));
+		define( 'DB_CHARSET', 'utf8' );
+		define( 'DB_COLLATE', '' );
+
+		// Debug mode should be disabled on Platform.sh. Set this constant to true
+		// in a wp-config-local.php file to skip this setting on local development.
+		if (!defined( 'WP_DEBUG' )) {
+			define( 'WP_DEBUG', false );
+		}
+
+		// Set all of the necessary keys to unique values, based on the Platform.sh
+		// entropy value.
+		// if ($config->projectEntropy) {
+		if (getenv('PLATFORM_PROJECT_ENTROPY')) {
+			$keys = [
+				'AUTH_KEY',
+				'SECURE_AUTH_KEY',
+				'LOGGED_IN_KEY',
+				'NONCE_KEY',
+				'AUTH_SALT',
+				'SECURE_AUTH_SALT',
+				'LOGGED_IN_SALT',
+				'NONCE_SALT',
+			];
+			$entropy = getenv('PLATFORM_PROJECT_ENTROPY');
```

Comment:
Like... exactly what this code is doing. This seems better than trying to set the env vars based on it as well.  Or am I missing something?